scripts/yaml-to-json.js

```
@@ -1,33 +1,33 @@
   }
 
   const newJsonData = {
-    $schema: "https://inlang.com/schema/inlang-message-format",
-    ...Object.entries(yamlData).reduce((acc, [key, value]) => ({
-      ...acc,
-      [key]: typeof value === 'string' ? value : JSON.stringify(value)
-    }), {})
+    $schema: 'https://inlang.com/schema/inlang-message-format',
+    ...Object.entries(yamlData).reduce(
+      (acc, [key, value]) => ({
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)